tests/test_classifier/test_rocauc.py

```diff
@@ -69,46 +51,20 @@ class FakeClassifier(BaseEstimator, ClassifierMixin):
 ##  Tests
 ##########################################################################
 
+@pytest.mark.usefixtures("binary", "multiclass")
 class ROCAUCTests(VisualTestCase, DatasetMixin):
 
-    def load_binary_data(self):
-        """
-        Returns the binary test data set.
-        """
-        # Load the Data
-        data = self.load_data("occupancy")
-
-        X = data[[
-            "temperature", "relative_humidity", "light", "C02", "humidity"
-        ]]
-
-        y = data['occupancy'].astype(int)
-
-        # Convert X to an ndarray
-        X = X.copy().view((float, len(X.dtype.names)))
-
-        # Return train/test splits
-        return tts(X, y, test_size=0.2, random_state=42)
-
-    def load_multiclass_data(self):
-        """
-        Returns the multiclass test data set.
-        """
-        raise NotImplementedError("Need to add multiclass data soon!")
-
     @pytest.mark.skip(reason="binary classifiers don't currently work as expected")
```

Comment:
If you have the chance while updating the Windows tests, would you mind changing this from `skip` to `xfail`?